src/components/policy/policy_regular/src/policy_table/types.cc

```diff
@@ -1354,7 +1354,6 @@ void ModuleMeta::ReportErrors(rpc::ValidationReport* report__) const {
 
 void ModuleMeta::SetPolicyTableType(PolicyTableType pt_type) {
   CompositeType::SetPolicyTableType(pt_type);
-  ccpu_version.SetPolicyTableType(pt_type);
```

Comment:
@ydementieiev are you sure that we dont need it here?

Reply:
@AByzhynar  this line was added by me before [here](https://github.com/LuxoftSDL/sdl_core/commit/309671f16072568421cf22f57d7747b78331f938), but after testing we see that it was not needed